client/my-sites/email/email-providers-stacked-comparison/provider-cards/get-on-submit-new-mailboxes-handler.ts

```diff
@@ -0,0 +1,86 @@
+import { ShoppingCartManagerActions } from '@automattic/shopping-cart';
+import page from 'page';
+import { canCurrentUserAddEmail, getCurrentUserCannotAddEmailReason } from 'calypso/lib/domains';
+import { ResponseDomain } from 'calypso/lib/domains/types';
+import { GOOGLE_PROVIDER_NAME } from 'calypso/lib/gsuite/constants';
+import { TITAN_PROVIDER_NAME } from 'calypso/lib/titan/constants';
+import { recordTracksEventAddToCartClick } from 'calypso/my-sites/email/email-providers-stacked-comparison/provider-cards/utils';
+import getCartItems from 'calypso/my-sites/email/form/mailboxes/components/utilities/get-cart-items';
+import { getMailProductProperties } from 'calypso/my-sites/email/form/mailboxes/components/utilities/get-mail-product-properties';
+import { MailboxOperations } from 'calypso/my-sites/email/form/mailboxes/components/utilities/mailbox-operations';
+import { EmailProvider } from 'calypso/my-sites/email/form/mailboxes/types';
+import { errorNotice } from 'calypso/state/notices/actions';
+import { ProductListItem } from 'calypso/state/products-list/selectors/get-products-list';
+
+export type GetOnSubmitNewMailboxesHandlerProps = {
+	comparisonContext: string;
+	domain: ResponseDomain;
+	dispatch: ( action: unknown ) => void;
+	emailProduct: ProductListItem | null;
+	isDomainInCart?: boolean;
+	provider: EmailProvider;
+	setAddingToCart: ( addingToCard: boolean ) => void;
```

Comment:
Since that's a boolean we should prefix that parameter:

```suggestion
	setAddingToCart: ( isAddingToCard: boolean ) => void;
```